hibernate-core/src/main/java/org/hibernate/boot/model/source/spi/AnyMappingSource.java

```diff
@@ -15,4 +15,8 @@
 public interface AnyMappingSource {
 	AnyDiscriminatorSource getDiscriminatorSource();
 	AnyKeySource getKeySource();
+
+	default boolean isLazy() {
+		return true;
```

Comment:
Do we need a default value here? Even if we do, wouldn't it be `false` since [`Any.fetch` defaults to `FetchType.EAGER`](http://docs.jboss.org/hibernate/orm/5.3/javadocs/org/hibernate/annotations/Any.html#fetch--)?

Reply:
Yes, we do since this was the previous behavior. If we change this, we will introduce a backward incompatible change.

Reply:
I see. But will you add a comment for indicating that it's for backward compatibility? Otherwise it would be confusing since it is different from the documentation.